cmd/zoekt-webserver/main.go

```diff
@@ -474,59 +476,14 @@ func traceID(ctx context.Context) string {
 
 // traceIDFromSpan returns a trace ID, if any, found in the given span.
 func traceIDFromSpan(span opentracing.Span) string {
-	spanCtx, ok := span.Context().(jaeger.SpanContext)
-	if !ok {
-		return ""
-	}
-	return spanCtx.TraceID().String()
-}
+	switch v := span.Context().(type) {
+	case jaeger.SpanContext:
+		return v.TraceID().String()
 
-func initializeJaeger() {
-	jaegerDisabled := os.Getenv("JAEGER_DISABLED")
-	if jaegerDisabled == "" {
-		return
-	}
-	isJaegerDisabled, err := strconv.ParseBool(jaegerDisabled)
-	if err != nil {
-		log.Printf("EROR: failed to parse JAEGER_DISABLED: %s", err)
-		return
-	}
-	if isJaegerDisabled {
-		return
-	}
-	cfg, err := jaegercfg.FromEnv()
-	cfg.ServiceName = "zoekt"
-	if err != nil {
-		log.Printf("EROR: could not initialize jaeger tracer from env, error: %v", err.Error())
-		return
-	}
-	cfg.Tags = append(cfg.Tags, opentracing.Tag{Key: "service.version", Value: zoekt.Version})
-	if reflect.DeepEqual(cfg.Sampler, &jaegercfg.SamplerConfig{}) {
-		// Default sampler configuration for when it is not specified via
-		// JAEGER_SAMPLER_* env vars. In most cases, this is sufficient
-		// enough to connect to Jaeger without any env vars.
-		cfg.Sampler.Type = jaeger.SamplerTypeConst
-		cfg.Sampler.Param = 1
+	case ddtrace.SpanContext:
+		return strconv.FormatUint(span.Context().(ddtrace.SpanContext).TraceID(), 10)
```

Comment:
you can also use v here.
```suggestion
		return strconv.FormatUint(v.TraceID(), 10)
```